instrumentation/src/main/java/io/opentelemetry/android/features/diskbuffering/scheduler/DefaultExportScheduleHandler.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.features.diskbuffering.scheduler
+
+import io.opentelemetry.android.internal.services.ServiceManager
+import io.opentelemetry.android.internal.services.periodicwork.PeriodicWorkService
+import java.util.concurrent.atomic.AtomicBoolean
+
+class DefaultExportScheduleHandler : ExportScheduleHandler {
+    private val enabled = AtomicBoolean(false)
+
+    override fun enable() {
+        if (!enabled.getAndSet(true)) {
+            ServiceManager.get().getService(PeriodicWorkService::class.java)
+                .enqueue(DefaultExportScheduler())
+        }
+    }
+
+    override fun disable() {
+        // No operation.
```

Comment:
Should we flip `enabled` here and stop the `ServiceManager` as well?

Reply:
We could flip `enabled`, although `ServiceManager` shouldn't be affected by the disk buffering behavior. Probably the name is a bit misleading, but `ServiceManager` is just a set of "services" (not all of them are constantly running, so wouldn't need to be "stopped") where we can access tools globally, especially tools related to Android OS APIs which tend to require an android `Context` and also different ways to use them depending on the device's API level... Which is annoying and difficult to maintain if we made those calls all over the project, so `ServiceManager` is a convenient place to have those tools available everywhere and protecting the rest of the code from future API breaking changes. It's also nice to have tools like `PeriodicWorkService` which allows to centralize any background work needed across multiple features, such as the disk buffering exports in this case, and in the future it might help with NTP calls as well. There's more info on ServiceManager [here](https://github.com/open-telemetry/opentelemetry-android/issues/129).

Reply:
Gotcha, `disable` can be a default method then (NoOp during the declaration), and it's only implemented if necessary.

Reply:
Thanks, good point. I've just made the changes.